src/napari_cryoet_data_portal/_filter.py

```diff
@@ -25,8 +25,7 @@ def load(self, client: Client) -> Generator[Tuple[Dataset, List[Tomogram]], None
         for dataset in Dataset.find(client, gql_filters):
             tomograms: List[Tomogram] = []
             for run in dataset.runs:
-                for spacing in run.tomogram_voxel_spacings:
-                    tomograms.extend(spacing.tomograms)
+                tomograms.extend(run.tomograms)
```

Comment:
we can also get all tomograms for a dataset directly, like this, which should be faster than iterating over all the runs (pardon the typos):
```
tomograms = Tomograms.find(client, Tomogram.run.dataset_id == dataset.id)
``